DFC.ServiceTaxonomy.GraphSync/DFC.ServiceTaxonomy.GraphSync.csproj

```diff
@@ -31,4 +32,8 @@
     <ProjectReference Include="..\DFC.ServiceTaxonomy.Neo4j\DFC.ServiceTaxonomy.Neo4j.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Views\Items\Events\" />
```

Comment:
Is this required? Looks like it's left over from event views once being in their own folder?